storage/database/db_manager.go

```diff
@@ -477,7 +477,11 @@ func (dbm *databaseManager) NewBatch(dbEntryType DBEntryType) Batch {
 
 func (dbm *databaseManager) getDBDir(dbEntry DBEntryType) string {
 	miscDB := dbm.getDatabase(MiscDB)
+
 	enc, _ := miscDB.Get(databaseDirKey(uint64(dbEntry)))
+	if len(enc) == 0 {
+		return dbDirs[dbEntry]
+	}
```

Comment:
I think we need to check wheter `enc` is `nil` or not. 
Or we can check and print an error message when `miscDB.Get()` failed.
```
enc, _ := miscDB.Get(databaseDirKey(uint64(dbEntry)))
if enc != nil && len(enc) != 0 {
	return string(enc)
}
return dbDirs[dbEntry]
```

Reply:
Checking error and logging the message sounds good.
However,  I think @aidan-kwon 's suggestion will make a difference. `len(enc) == 0` returns true for both when enc is nil and empty byte.

Reply:
@winnie-byun You're right. It was my mistake. 
I will resolve this conversation.